tests/mpls_input/appl_db.json

```diff
@@ -26,22 +23,10 @@
     "INTF_TABLE:Ethernet8:192.168.8.1/24": {
         "NULL": "NULL"
     },
-    "INTF_TABLE:Loopback0": {
-        "NULL": "NULL"
-    },
-    "INTF_TABLE:Loopback0:192.168.0.1/24": {
-        "NULL": "NULL"
-    },
-    "INTF_TABLE:PortChannel2": {
+    "INTF_TABLE:Ethernet12": {
         "mpls": "disable"
     },
-    "INTF_TABLE:PortChannel2:10.0.0.56/31": {
-        "NULL": "NULL"
-    },
-    "INTF_TABLE:Vlan2": {
+    "INTF_TABLE:Ethernet20": {
```

Comment:
In case you're wondering why some of the ports from the previous version do not appear here, it is because those ports do not appear in mock_tables/configdb